### PR TITLE
Fix(CSS): `outline` doesn't handle some keyword values

### LINKED ISSUE #296: 🐞 `outline:none` is broken
### Description

`outline:*` is always rewrited in css as `outline:* solid`. Because of this behavior there is impossible to use `outline:none` because this is translated to `outline:none solid` what is invalid property value.

There is also no unit tests cover that case.

### Reproduction

1. Add class `outline:none` to the input.
2. See the broken css in dev tools.

### System Informations

N/A

### PR DESCRIPTION
**Pull request by lesha1201**
Now `outline` works well with CSS-wide keywords and all `outline-style` keywords.

Fixes https://github.com/master-co/css/issues/296

List of CSS-wide keywords:

- https://drafts.csswg.org/css-values-4/#css-wide-keywords

`outline` spec:

- https://drafts.csswg.org/css-ui/#outline-props

`outline-style` spec:

- https://drafts.csswg.org/css-ui/#propdef-outline-style

### CODE REVIEW

File: packages/css/src/config/rules.ts [hunk 1]
```diff
@@ -1668,9 +1668,15 @@ const rules = {
         layer: CoreLayer.NativeShorthand,
         colored: true,
         transform(value) {
-            if (!/hidden|dotted|dashed|solid|double|groove|ridge|inset|outset/i.test(value)) {
+            const outlineGlobalValuesRegex =
+                /^(?:inherit|initial|revert|revert-layer|unset)$/i
```
Comment (lesha1201) [inline packages/css/src/config/rules.ts:1672]: These values are applicable for all CSS properties but decided to keep it here while it's used only in this place.

**Comment by 1aron** [comment]
@lesha1201 You move fast, and I was just about to fix that issue 😂

**Comment by lesha1201** [comment]
> @lesha1201 You move fast, and I was just about to fix that issue 😂

@1aron Sorry, didn't notice an assignee. Feel free to edit or close this PR as you want.

**Comment by 1aron** [comment]
@lesha1201 No, you did a good job, and I'll merge it as soon as possible and make some tweaks 💯

**Comment by 1aron** [comment]
@lesha1201 I'm considering removing the behavior of adding `solid` to avoid these side effects.

And solve the problem of presetting `border` in `@master/normal.css`: https://github.com/master-co/css/blob/beta/packages/normal.css/src/index.css#L23-L27

```diff
*,
::before,
::after {
    box-sizing: border-box;
+   border-style: solid;
}
```
This way, we also get minimal CSS output without `solid`, for example `border:16|black`:
```css
.border\:16\|black {
    border: 1rem rgb(0 0 0)
}
```

**Comment by 1aron** [comment]
- #267

@lesha1201 The final solution may be to preset the outline and border styles in `@master/normal.css` and remove the auto-fill behavior for related rules.

```css
*,
::before,
::after {
    border: 0 solid;
    outline: 0 solid;
}
```
If you want to keep accessibility: 
```css
:focus {
  outline: 2px solid blue;
  outline-offset: 2px;
}
```
Applying that approach will result in less CSS generated.

**Comment by lesha1201** [comment]
@1aron It's ok to have it for `border` but I'm really unsure about `outline`. Disabling it by default migth decrease accessibility significantly because many devs won't be paying attention to it and just will keep the default behavior without realizing that accessibility in this case is worse.

By the way, modern browsers uses `:focus-visible` by default. They also use a special double color that works with any background color (dark/light).

**Comment by 1aron** [comment]
@lesha1201 Yep, the default outline styles for most elements:

image

If we could make sure that the above is the default behavior of all elements, then just reset focus outline again in  **@master/normal.css**:
```css
*,
::before,
::after {
    border: 0 solid;
    outline: 0 solid;
}

:focus-visible {
    outline: -webkit-focus-ring-color auto 1px;
}
```

**Pull request merged**